app/models/author.rb

```diff
@@ -0,0 +1,14 @@
+# A person who write a book.
```

Comment:
## Reek Code Smells Found
[UtilityFunction](https://github.com/troessner/reek/blob/master/docs/Utility-Function.md): doesn't depend on instance state (maybe move it to another class?)
Lines: [9]